src/components/modules/epics/ContributionsEpicArticleCountOptOut.tsx

```diff
@@ -0,0 +1,295 @@
+// --- Imports --- //
+
+import React, { useState } from 'react';
+import { body, textSans } from '@guardian/src-foundations/typography';
+import { palette, space } from '@guardian/src-foundations';
+import { Button } from '@guardian/src-button';
+import { ButtonLink } from '@guardian/src-link';
+import { css } from '@emotion/core';
+import { OphanComponentEvent } from '../../../types/OphanTypes';
+import {
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_OUT_OPEN,
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_OUT_CLOSE,
+    OPHAN_COMPONENT_ARTICLE_COUNT_STAY_IN,
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_OUT,
+    OPHAN_COMPONENT_ARTICLE_COUNT_STAY_OUT,
+    OPHAN_COMPONENT_ARTICLE_COUNT_OPT_IN,
+} from './utils/ophan';
+
+// --- Styles --- //
+
+const containerStyles = css`
+    ${body.medium({ fontWeight: 'bold' })};
+    font-style: italic;
+`;
+
+const optOutContainer = css`
+    color: ${palette.opinion[400]};
+`;
+
+const articleCountOnHeaderContainerStyles = css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+`;
+
+const articleCountTextStyles = css`
+    ${textSans.small()};
+    margin-right: ${space[1]}px;
+`;
+
+const articleCountWrapperStyles = css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-right: ${space[2]}px;
+    justify-content: flex-end;
+`;
+
+const articleCountCtaStyles = css`
+    ${textSans.small({ fontWeight: 'bold' })};
+`;
+
+const articleCountDescriptionTopContainerStyles = css`
+    border-top: 1px solid #000000;
```

Comment:
this is only a tiny one, but we can use `neutral[0]` here from the design systems. It doesn't really make much of a difference, but it's quite nice to reduce the number of hex strings floating about